_authors/arbel.md

```diff
@@ -1,3 +1,6 @@
 ---
 title: "Keren Arbel"
 ---
+Keren has been immersed in Buddhist practice since 1997, practicing with variety of teachers in Asia and the West. Keren also holds a PhD in Buddhist Studies.
+
+Keren teaches workshops and retreats on mindfulness and insight meditation (vipassana), loving kindness  (metta) and Buddhist wisdom at the Israeli Insight Meditation Society [(“Tovana”)](https://www.tovana.org.il/en). She is a co-founder and a teacher at [(“Nalanda“)](https://www.nalanda.org.il/vision-en): an Israeli Buddhist institute dedicated to contemplative study of the Buddhist path. She also teaches in the MBSR teacher training program and the MBCT Facilitation Certificate at IDC Herzliya’s [Muda](https://muda.idc.ac.il/wp-content/uploads/2020/08/Mindfulness-Based-Social-Resilience-Platform-Muda-Aug2020-1.pdf) Institute for Mindfulness, Science and Society. For more than ten years Keren taught Buddhist philosophy and psychology as an adjunct professor in the Department of East Asian Studies at Tel Aviv University. From 2017 Keren dedicates all her time to teaching the Dhamma.
```

Comment:
Perhaps "Dr. Arbel" for the first use in this paragraph, then "she" after until the last one.

`[(“Tovana”)]` no need for the `“”`s

`(“Nalanda“)` no need for the quotes or `()`s

`From 2017 Keren` should be `Since 2017, Keren`

Also: citation needed at the end.